st2debug/st2debug/cmd/submit_debug_info.py

```diff
@@ -234,10 +294,12 @@ def create_archive(include_logs, include_configs, include_content, include_syste
         'logs': os.path.join(temp_dir_path, 'logs/'),
         'configs': os.path.join(temp_dir_path, 'configs/'),
         'content': os.path.join(temp_dir_path, 'content/'),
+        'commands': os.path.join(temp_dir_path, 'commands/'),
         'system_info': os.path.join(temp_dir_path, 'system_info.yaml'),
         'user_info': os.path.join(temp_dir_path, 'user_info.yaml')
     }
-
+    if config_yaml:
+        DIRECTORY_STRUCTURE.append('commands/')
```

Comment:
Can we just add 'commands/' to the list defined at line 97? Any downside to having it there all the time?